app/views/contexts/calculated_summary_context.py

```diff
@@ -87,12 +85,12 @@ def build_view_context_for_calculated_summary(
         }
 
     def _build_calculated_summary_section(
-        self, rendered_block: ImmutableDict, current_location: Location
+        self, rendered_block: Dict, current_location: Location
     ) -> Mapping:
```

Comment:
From our type hints guide: "Make return types as specific as possible (to be predictable to callers)"  and "Specify at least the top-level type parameters".

Reply:
Updated